TM1py/Services/CellService.py

```diff
@@ -680,12 +702,24 @@ def write_through_unbound_process(self, cube_name: str, cellset_as_dict: Dict, i
                 else:
                     value_str = format(value, f'.{precision}f')
 
+            comma_separated_elements = ",".join("'" + element.replace("'", "''") + "'" for element in coordinates)
+
+            if skip_non_updateable:
+                cell_is_updateable_pre = f"IF(CellIsUpdateable('{cube_name}', {comma_separated_elements})=1,"
+                cell_is_updateable_post = ",0);"
+            else:
```

Comment:
why not make the statements in else condition default? (saves a line of code).

Reply:
OK